packages/synapse-interface/components/_Transaction/_Transaction.tsx

```diff
@@ -208,7 +208,7 @@ export const _Transaction = ({
                 text="Contact Support"
                 link="https://discord.gg/synapseprotocol"
               />
-              {isTxComplete && (
+              {isTxCompleted && (
```

Comment:
for this one can we rename `isTxCompleted` to `isTxFinalized` ? just confusing bc there are both `isTxCompleted` and `isTxComplete` & want to make it readable quickly when next revisiting